test/unit/app/presenters/publishing_api/editionable_worldwide_organisation_presenter_test.rb

```diff
@@ -0,0 +1,38 @@
+require "test_helper"
+
+class PublishingApi::EditionableWorldwideOrganisationPresenterTest < ActiveSupport::TestCase
+  def present(...)
+    PublishingApi::EditionableWorldwideOrganisationPresenter.new(...)
+  end
+
+  test "presents a Worldwide Organisation ready for adding to the publishing API" do
+    worldwide_org = create(:editionable_worldwide_organisation)
+
+    public_path = worldwide_org.public_path
+
+    expected_hash = {
+      base_path: public_path,
+      title: worldwide_org.title,
+      schema_name: "worldwide_organisation",
+      document_type: "editionable_worldwide_organisation",
```

Comment:
Has this `document_type` been defined in Publishing API schemas?

Reply:
https://github.com/alphagov/publishing-api/pull/2578